import-scripts/knowledgesystems-importer/automation-environment.sh

```diff
@@ -52,16 +52,7 @@ export_java_home "AUTODETECT"
 export JAVA_PROXY_ARGS="-Dhttp.proxyHost=jxi2.mskcc.org -Dhttp.proxyPort=8080 -Dhttp.nonProxyHosts=draco.mskcc.org|pidvudb1.mskcc.org|phcrdbd2.mskcc.org|dashi-dev.cbio.mskcc.org|pipelines.cbioportal.mskcc.org|localhost"
 # the legacy environment variable "JAVA_BINARY" is maintained for older personal scripts. All production scripts now use JAVA_8_BINARY or JAVA_11_BINARY when launching java
 export JAVA_BINARY="$JAVA_HOME/bin/java"
-export JAVA_8_HOME="/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.342.b07-1.amzn2.0.1.x86_64"
-export JAVA_8_BINARY="$JAVA_8_HOME/jre/bin/java"
-export JAVA_19_HOME="/usr/lib/jdk-19.0.2"
-export JAVA_19_BINARY="$JAVA_19_HOME/bin/java"
-export JAVA_BINARY_FOR_CRDB="$JAVA_8_BINARY"
-export JAVA_BINARY_FOR_CVR="$JAVA_19_BINARY"
-export JAVA_BINARY_FOR_DARWIN="$JAVA_8_BINARY"
-export JAVA_BINARY_FOR_DDP="$JAVA_8_BINARY"
-export JAVA_BINARY_FOR_REDCAP="$JAVA_19_BINARY"
-export JAVA_BINARY_FOR_IMPORTER="$JAVA_8_BINARY"
+export JAVA_HOME="/usr/lib/jvm/java-11-openjdk-11.0.20.0.8-1.amzn2.0.1.x86_64"
```

Comment:
wouldn't this need to be moved above line 54? (which depends on JAVA_HOME?)
The functions above this detect JAVA_HOME based on the location of java with the command "which java". Another way to manage this is to delete line 55 and adjust /home/cbioportal_importer/.bashrc so that the path to the java 11 jvm comes earlier in the PATH environment than any other JVM path. Note that this should be tested for both interactive shells and non-interactive shells (such as those launched by crontab).